_posts/fr/2022-02-03-project-a-produit.md

```diff
@@ -0,0 +1,243 @@
+---

```

Comment:
I can't review the rest of the french but wondering...should the filename be projet-a-produit? (remove the 'c')

Reply:
yeeeaaaahhh... got lazy there. It's an md file so in the background :P The html file will still be english as it's how Jekyll switches between french and english.

Reply:
The filename shows up in the URL. Isn't it an OL rule that French websites must have french URLs? In order to switch between french and english the metadata 'ref' has to be the same but the filenames can be different.